modules/Makefile.circleci

```diff
@@ -45,6 +45,8 @@ circle\:tag-latest:
 	@$(SELF) docker:login
 	@echo "INFO: Tagging latest"
 	@$(SELF) DOCKER_TAG=latest docker:push
+	@git tag --force "$(CIRCLE_BRANCH)-docker-latest"
```

Comment:
No, in a `Makefile`, every line not terminated with a `\` is a separate command. Any command that exits non-zero causes an immediate failure from that point. So in this case, if the `docker:push` fails, the `git tag` will never be run.